keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/BakedImage.kt

```diff
@@ -10,7 +10,7 @@ data class BakedImage(
   val vmType: String,
   val cloudProvider: String,
   val appVersion: String,
-  val baseAmiVersion: String,
+  val baseAmiName: String,
```

Comment:
You'll need to make the corresponding change in the [bakeryLifecycleMonitor](https://github.com/spinnaker/keel/blob/master/keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/artifact/BakeryLifecycleMonitor.kt#L120), which means changing the function called there in [ImageService](https://github.com/spinnaker/keel/blob/master/keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageService.kt#L164).